README.adoc

```diff
@@ -36,3 +36,6 @@ These are optional settings that can be manually added to the page URL to contro
  ** Disable comment count loading because it's a bit expensive. e.g. `comments=false`
  * `autocomments`
  ** Enable auto comments when making changes to bugs. e.g. `autocomment=true`
+ * `picture`
+ ** Use a picture site option.
+ 1. gravatar
```

Comment:
I wonder if we are likely to support other sources? I think gravatar is the only (popular) one that takes an email address and returns a photo. We can probably simplify this to `gravatar=true` eh?